src/app/caesar-portal/locations/locations.component.html

```diff
@@ -0,0 +1,17 @@
+<p (click)="openModal(template)">Locations<span class="pull-right glyphicon glyphicon-globe"></span></p>
+ 
+<ng-template #template>
+    <div class="modal-body">
+        <div class="row">
+            <div class="col-md-4 " *ngFor="let item of locations">
+                <div class="location" (click)="addLocation($event, item.name.toLowerCase())" (dblclick)="addAndConfirmLocation(item.name.toLowerCase())">
```

Comment:
you can split each property on new line for better readability

Reply:
ok